include/geometry.h

```diff
@@ -23,6 +23,13 @@ class Geometry {
   Eigen::Matrix<double, Tdim, Tdim> inverse_rotation_matrix(
       const Eigen::Matrix<double, Tdim, 1>& angles) const;
 
+  //! Return angle between two vectors in radians
+  //! \param[in] vector_a First vector
+  //! \param[in] vecotr_b Second vector
+  //! \retval anti-clockwise angle from vector_a to vector_b
+  double angle_between_two_vectors(const Eigen::VectorXd& vector_a,
```

Comment:
Why can't it be `Vector<double, Tdim, 1>`?

Reply:
We can do that too